java/vector/src/main/codegen/templates/AbstractPromotableFieldWriter.java

```diff
@@ -287,6 +298,11 @@ public ListWriter list(String name) {
     return getWriter(MinorType.STRUCT).list(name);
   }
 
+  @Override
+  public ListWriter listView(String name) {
+    return getWriter(MinorType.STRUCT).listView(name);
```

Comment:
Why is this struct? (Also isn't it a duplicate of above?)

Reply:
So the capability for list is in the above, and the newly added is for listview. Is it inaccurate?